specification/app/resource-manager/Microsoft.App/preview/2023-08-01-preview/Builds.json

```diff
@@ -388,6 +393,23 @@
         "server"
       ]
     },
+    "BuildEndpoint": {
+      "type": "object",
+      "properties": {
+        "endpoint": {
+          "type": "string",
+          "description": "Endpoint url."
+        },
+        "token": {
+          "type": "string",
```

Comment:
Use x-ms-secret here.

Reply:
@visingla-ms good catch, thanks. Updated!

Reply:
sorry, i see what you mean by post for secrets. will make that update shortly

Reply:
Yeah, please fix the examples which are returning secret value in non-POST operations. And, update your service backend implementation to take care of that as well.

Reply:
Service backend isn't released yet as this is a new feature, but will make sure that the implementations match the defined API here. Thanks

Reply:
I believe this has been completed, however I'm waiting for the pipelines to run now. Thanks for bringing this up!